scripts/CSSRE/rolling-restart-brokers/metadata.yaml

```diff
@@ -0,0 +1,43 @@
+file: rolling-restart-brokers.sh
+name: rolling-restart-brokers
+description: Manually roll the Kafka brokers
+author: Pat Cremin
+allowedGroups:
+  - CSSRE
+rbac:
+  clusterRoleRules:
+    - verbs:
+        - "get"
+        - "list"
+      apiGroups:
+        - "apps"
+      resources:
+        - "statefulsets"
+    - verbs:
+        - "get"
+        - "list"
+        - "patch"
+      apiGroups:
+        - ""
+      resources:
+        - "pods"
+    - verbs:
+        - "list"
+      apiGroups:
+        - "kafka.strimzi.io"
+      resources:
+        - "kafkas"
+    -verbs:
```

Comment:
Hey @pat-cremin, I think this is where the pipeline is failing, please add space between `- verbs`